drivers/mdio/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
The commit `mdio: Add shell support` contains the actual driver. Please make the shell and the driver to separate commits.